docs/plugins.md

```diff
@@ -13,10 +13,10 @@ Let's take a quick look at the current dokku nginx plugin that's shipped with do
 ## Installing a plugin
 
 ```shell
-cd /var/lib/dokku/plugins/available
-git clone <git url>
-dokku plugin:install
+dokku plugin:install <git url>
 ```
+Note: `plugin:install` might trigger actions like `apt-get`, please make sure you inspect any potential plugin's `install` or `install-dependencies` trigger, and (of course) execute this command as the `root` user
+
```

Comment:
Not necessarily. You could clone it if you run `plugn install` as the `dokku` user. However `plugn trigger install` will fail with the reported error.

```
root@dokku:/var/lib/dokku# ls -lad ./*/{available,enabled}
drwxr-xr-x 21 dokku dokku 4096 Sep 19 00:27 ./core-plugins/available
drwxr-xr-x  2 dokku dokku 4096 Sep 19 00:27 ./core-plugins/enabled
drwxr-xr-x  2 dokku dokku 4096 Sep 22 16:01 ./plugins/available
drwxr-xr-x  2 dokku dokku 4096 Sep 22 16:01 ./plugins/enabled
```